jabbercat/models.py

```diff
@@ -748,13 +747,13 @@ class RosterFilterModel(Qt.QSortFilterProxyModel):
     def __init__(self, parent: Qt.QObject=None):
         super().__init__(parent)
 
-        self._tags_filter_model = None
-        self._tags_filter_set = None
-        self._tags_filter_set_connections = []
-        self._filter_by_text = None
+        self._tags_filter_model: Qt.QAbstractItemModel = None
```

Comment:
Do not use this syntax. It is not compatible with Python 3.5, which is the Python version we are (currently) aiming for. There is a surrogate syntax based on comments I think.

Reply:
OK